client/dune

```diff
@@ -4,3 +4,10 @@
   (modules     dns_client)
   (libraries   dns.cache domain-name dns randomconv rresult)
   (wrapped     false))
+
+(library
```

Comment:
```suggestion

```

Reply:
What is the suggestion here, I don't think I understand?

Reply:
to remove the dns_client_tls library which causes build failures, sorry for the obfuscated github comments.  and thanks for pushing such a commit which removes that anyways! :)